modules/extensions-xml-support/src/main/java/org/mule/runtime/extension/internal/processor/ModuleOperationMessageProcessor.java

```diff
@@ -162,7 +162,9 @@ public ModuleOperationMessageProcessor(Map<String, String> properties,
 
     for (ParameterModel parameterModel : parameterModels) {
       final String parameterName = parameterModel.getName();
-      if (parameters.containsKey(parameterName)) {
+      if (parameterName.equals("target") || parameterName.equals("targetValue")) {
```

Comment:
Don't we have a constant somewhere for this?

Reply:
i searched for it. The answer will surprise you.

Reply:
org.mule.runtime.extension.api.ExtensionConstants#TARGET_PARAMETER_NAME